src/scene/sprite-atlas.js

```diff
@@ -0,0 +1,34 @@
+pc.extend(pc, function () {
+    'use strict';
+
+    /**
+    * @name pc.SpriteAtlas
```

Comment:
This should be TextureAtlas not SpriteAtlas

Reply:
Good point, @daredevildave. I do agree. However, a texture atlas is quite a generic thing. Is the SpriteAtlas class currently geared just towards sprites? Looks like it is. Does this object belong in the src/graphics folder? Or src/scene? Comments @guycalledfrank?

Reply:
It's pretty general purpose, it defines a Texture that has uv rects defined.

Reply:
So it is 'kinda' general purpose. A frame has uv's but it also have pivots which are unrelated outside the context of sprites. That's why I named it SpriteAtlas. We could use it for regular materials too I guess but then we'd also store the pivot which is probably not needed.